examples/jenkins/pipeline/README.md

```diff
@@ -0,0 +1,60 @@
+This set of files will allow you to deploy a Jenkins server that is capable of executing Jenkins pipelines and
+utilize pods run on OpenShift as Jenkins slaves.
+
+To walk through the example:
+1) Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
+
+https://github.com/openshift/origin
```

Comment:
since your are talking specifically about the imagestreams, should this URL include `examples/image-streams/` at the end of it, possibly list the two json files as well?